tests/ProxyManagerTest/GeneratorStrategy/FileWriterGeneratorStrategyTest.php

```diff
@@ -27,17 +30,32 @@
  *
  * @group Coverage
  * @covers \ProxyManager\GeneratorStrategy\FileWriterGeneratorStrategy
+ * @runTestsInSeparateProcesses
  *
  * Note: this test generates temporary files that are not deleted
  */
 class FileWriterGeneratorStrategyTest extends TestCase
 {
+    /** @var string */
+    private $tempDir;
+
+    protected function setUp() : void
+    {
+        $this->tempDir = sys_get_temp_dir() . '/' . self::class;
```

Comment:
Consider using `uniqid()` or similar, instead of using the class name (which, by the way, includes backslashes: won't work on windows)

Reply:
Right, used `uniqid()` instead.